src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const images = [...document.querySelectorAll('a')];
```

Comment:
The `images` variable is collecting all anchor (`<a>`) elements, not images. If the intention is to collect images, you should use a selector like 'img' instead.